source/common/filesystem/filesystem_impl.cc

```diff
@@ -19,6 +17,7 @@
 
 #include "common/common/assert.h"
 #include "common/common/thread.h"
+#include "common/filesystem/filesystem_impl.h"
```

Comment:
Google include order has a special case for foo.cc including foo.h as the first line, so I think you can revert most of the include order changes.  

I think the fix_format script will catch this for you, so it's worth running it locally and seeing what else it turns up!